ArduCopter/mode_follow.cpp

```diff
@@ -106,8 +102,8 @@ void ModeFollow::run()
         const float des_vel_z_max = copter.avoid.get_max_speed(pos_control->get_pos_z_p().kP().get(), pos_control->get_max_accel_z() * 0.5f, fabsf(dist_vec_offs_neu.z), copter.G_Dt);
         desired_velocity_neu_cms.z = constrain_float(desired_velocity_neu_cms.z, -des_vel_z_max, des_vel_z_max);
 
-        // get avoidance adjusted climb rate
-        desired_velocity_neu_cms.z = get_avoidance_adjusted_climbrate(desired_velocity_neu_cms.z);
+        // limit the horizontal velocity to prevent fence violations
```

Comment:
we can remove the "horizontal" from this comment because I think it's adjusting the 3D velocity vector isn't it?